docker-compose.yaml

```diff
@@ -86,7 +86,7 @@ services:
     image: openzaak/open-zaak:1.8.2
     platform: linux/amd64
     environment:
-      - ALLOWED_HOSTS=localhost,host.docker.internal,openzaak.local
+      - ALLOWED_HOSTS=localhost,host.docker.internal,openzaak.local,example.com
```

Comment:
Why is example.com added here? example.com is not a real domain. it's only meant for dummy testing purposes.

Reply:
because i was not able to make a export without it (locally)

Reply:
You mean the export of the SQL insert statements from your local Open Zaak database in your database tool of choice? Weird that that would require `example.com` here..